pkg/cable/libreswan/libreswan.go

```diff
@@ -584,3 +607,55 @@ func (i *libreswan) runPluto() error {
 
 	return nil
 }
+
+func (i *libreswan) addClampMssToPathMTU(subnets string) {
+	ruleSpecSource := []string{"-s", subnets, "-p", "tcp", "-m", "tcp", "--tcp-flags", "SYN,RST", "SYN", "-j",
+		"TCPMSS", "--clamp-mss-to-pmtu"}
+	ruleSpecDest := []string{"-d", subnets, "-p", "tcp", "-m", "tcp", "--tcp-flags", "SYN,RST", "SYN", "-j",
+		"TCPMSS", "--clamp-mss-to-pmtu"}
+
+	if err := i.updateMangleTableEntryWithRetry(true, ruleSpecSource...); err != nil {
+		klog.Errorf("error appending iptables rule \"%s\": %v\n", strings.Join(ruleSpecSource, " "), err)
+	}
+
+	if err := i.updateMangleTableEntryWithRetry(true, ruleSpecDest...); err != nil {
+		klog.Errorf("error appending iptables rule \"%s\": %v\n", strings.Join(ruleSpecDest, " "), err)
+	}
+}
+
+func (i *libreswan) removeClampMssToPathMTU(subnets string) {
+	ruleSpecSource := []string{"-s", subnets, "-p", "tcp", "-m", "tcp", "--tcp-flags", "SYN,RST", "SYN", "-j",
```

Comment:
If I remember correctly, @mangelajo wanted to verify the behavior once why PMTU is not working (aka ICMP packets are not delivered). Are we planning to get this PR in or wait for @mangelajo to update his findings?

Another thing we discussed during the call was to check if this is applicable to all the CableDrivers?

Anyways, we also need a similar fix in submariner-operator code as we are using iptables for the first time in gateway pod - https://github.com/submariner-io/submariner-operator/commit/0448fcd60972da03c84c345c24c2cf1c34b6b42b

Reply:
> If I remember correctly, @mangelajo wanted to verify the behavior once why PMTU is not working (aka ICMP packets are not delivered). Are we planning to get this PR in or wait for @mangelajo to update his findings?

Maybe since the d/s release is nearing we can try this fix as this should be harmless. @mangelajo @nyechiel thoughts?
> 
> Another thing we discussed during the call was to check if this is applicable to all the CableDrivers?
I think the idea was to see if this fix works in downstream CI before porting it to other cable drivers.
> 
> Anyways, we also need a similar fix in submariner-operator code as we are using iptables for the first time in gateway pod - [submariner-io/submariner-operator@0448fcd](https://github.com/submariner-io/submariner-operator/commit/0448fcd60972da03c84c345c24c2cf1c34b6b42b)

I will check and raise a PR if required.